### PR TITLE
Expect "Type: " label prefix and ignore improvement labels

### PR DESCRIPTION
**Pull request by adamralph**
re #2

### CODE REVIEW

**Comment by adamralph** [comment]
@Particular/engineers please review. Our automated release notes will no longer work until this change has been made.

Related to https://github.com/Particular/GitHubReleaseNotes/issues/26

**Comment by johnsimons** [comment]
@adamralph this looks good to me.

Just a question, why do we have `Feature` and also `Improvement` ? Do we need both?

**Comment by adamralph** [comment]
@johnsimons actually we decided to remove support for the `Improvement` label. It's already gone from the GitHub repos, and this PR removes support for it in the code.

We decided 'improvements' and 'features' was an unnecessary separation and that 'improvements' were actually just less important features.

File: src/Engineering.ReleaseManagement/ReleaseNotesGenerator.cs [hunk 1]
```diff
@@ -7,6 +7,11 @@ namespace Engineering.ReleaseManagement
 {
     public class ReleaseNotesGenerator
     {
+        public static string LabelPrefix
```
Comment (SzymonPobiega) [inline src/Engineering.ReleaseManagement/ReleaseNotesGenerator.cs]: why not `const string`?

File: src/Engineering.Email/ReleaseAnnouncementIssuesContentProvider.cs [hunk 2]
```diff
@@ -25,6 +25,11 @@ public ReleaseAnnouncementIssuesContentProvider(IDocumentSetRepository documentS
             this.releaseDataAccess = releaseDataAccess;
         }
 
+        public static string LabelPrefix
```
Comment (SzymonPobiega) [inline src/Engineering.Email/ReleaseAnnouncementIssuesContentProvider.cs]: why not `const string`?

File: src/Engineering.ReleaseManagement/ReleaseNotesGenerator.cs (hunk 1 above)
Comment (adamralph) [inline src/Engineering.ReleaseManagement/ReleaseNotesGenerator.cs]: It's public, and it's not logically constant.

In fact, Eric Lippert argues that the latter condition alone is sufficient for not using `const` - see his comments on http://stackoverflow.com/questions/8140142/is-a-class-wide-integer-that-is-only-assigned-at-the-start-thread-safe-for-readi

File: src/Engineering.Email/ReleaseAnnouncementIssuesContentProvider.cs (hunk 2 above)
Comment (adamralph) [inline src/Engineering.Email/ReleaseAnnouncementIssuesContentProvider.cs]: See https://github.com/Particular/Automation.Engineering/pull/1/files#r39139439

File: src/Engineering.ReleaseManagement/ReleaseNotesGenerator.cs (hunk 1 above)
Comment (SzymonPobiega) [inline src/Engineering.ReleaseManagement/ReleaseNotesGenerator.cs]: ACK

**Pull request merged**